packager/packager.js

```diff
@@ -34,6 +34,10 @@ var parseCommandLine = require('./parseCommandLine.js');
 var webSocketProxy = require('./webSocketProxy.js');
 
 var options = parseCommandLine([{
+  command: 'origin',
+  default: 'http://localhost',
+  type: 'string',
+}, {
```

Comment:
Yes! `protocol`, `hostname`, and `port` as three separately configurable fields. Could even use Node's `url.format()` to construct the complete origin string if that helps.